accessibility-checker.php

```diff
@@ -525,11 +571,14 @@ function edac_output_simplified_summary( $content ) {
 	}
 	$simplified_summary          = edac_simplified_summary_markup( get_the_ID() );
 	$simplified_summary_position = get_option( 'edac_simplified_summary_position', $default = false );
-	if ( $simplified_summary && 'before' === $simplified_summary_position ) {
-		return $simplified_summary . $content;
-	}
-	if ( $simplified_summary && 'after' === $simplified_summary_position ) {
-		return $content . $simplified_summary;
+
+	if ( $simplified_summary ) {
+		if ( 'before' === $simplified_summary_position ) {
+			return $simplified_summary . $content;
+		}
+		if ( 'after' === $simplified_summary_position ) {
+			return $content . $simplified_summary;
+		}
 	}
```

Comment:
Since both conditions depend on `$simplified_summary`, they were grouped/nested to make it clearer.